pandas/core/base.py

```diff
@@ -668,7 +668,7 @@ class IndexOpsMixin(object):
     __array_priority__ = 1000
 
     def transpose(self, *args, **kwargs):
-        """ return the transpose, which is by definition self """
+        """ Return the transpose, which is by definition self """
```

Comment:
This is orthogonal to the change right? Would be better to keep these in a separate PR focused on fixing up these docstrings, especially given there are other issues with the docstrings (ex: missing period)

Reply:
Yes, sorry about that, this got merged from the other branch I've been working on.

I added a revert commit, but I'm going to look for a way of removing the commit entirely

Reply:
I've tried to do a rebase removing the commits from local tree that I then force-pushed to remote. That didn't seem to remove offending commits from PR.

@WillAyd do you mind if I create a new pull request once the rest of the issues have been rectified?